src/ssl.c

```diff
@@ -871,6 +871,10 @@ int wolfSSL_get_ciphers(char* buf, int len)
 
     /* Add each member to the buffer delimited by a : */
     for (i = 0; i < ciphersSz; i++) {
+#ifndef NO_CIPHER_SUITE_ALIASES
+        if (ciphers[i].flags & WOLFSSL_CIPHER_SUITE_FLAG_NAMEALIAS)
```

Comment:
if `-DNO_CIPHER_SUITE_ALIASES`, then `WOLFSSL_CIPHER_SUITE_FLAG_NAMEALIAS` is always false, so the check logic would just waste footprint and cycles.  The idea is to have `-DNO_CIPHER_SUITE_ALIASES` gate out any wasting of resources on the alias facility.

Reply:
Aaaah, I'm readin' ya.  Your question is, should the `buf` returned by `wolfSSL_get_ciphers()` include the alias names.  I think so, now that you've prompted me to think more deeply about it.

So with the latest push, the `continue` clause is gone.

This will cause no big surprises, as the alias names will of course function just like the canonical names when passed back to `wolfSSL_get_cipher_suite_from_name()`.  That's the whole point after all.  There will be a small surprise, in that if the ID returned by `wolfSSL_get_cipher_suite_from_name()` is passed back to `wolfSSL_get_cipher_name_from_suite()`, for aliases the returned name won't match the supplied name, but instead the canonical name is returned.  That's down-in-the-weeds stuff inherent to having aliases, that normal application code won't notice.

In the final analysis, I think indeed it's more valuable for `wolfSSL_get_ciphers()` to include all recognized names, so that callers performing their own textual comparisons to prequalify cipher names will get results consistent with those of callers that use the alias variants based on out-of-band knowledge of their existence.